### DISCUSSION TITLE
Welcome to APP-todolist-Grupal-Final Discussions!

### DISCUSSION THREAD

**Discussion by AlbertoSH2021** [AlbertoSH2021/APP-todolist-Grupal-Final#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlbertoSH2021** [comment]
Hola sobre los resultados del proyecto, deseo si alguien detecto el error que se genera al guardar loc cambios de editar permisos

**Comment by 201520432d** [comment]
Falta poner include conexión y una condicional if else para determinar el sesion star luego al pulsar guardar debemos re dirigir la página a el panel de administrador

**Reply by AlbertoSH2021** [reply]
es verdad, faltaba llamar la funcion de inicio de sesion

**Comment by AlbertoSH2021** [comment]
es verdad, faltaba llamar la funcion de inicio de sesion
